doc/nrf/config_and_build.rst

```diff
@@ -29,6 +29,8 @@ Depending on the board you are working with, check also its :ref:`hardware guide
 
 Make sure to consider :ref:`app_bootloaders` and :ref:`app_dfu` already at this stage of the application development.
 
+If you want to go through dedicated training related to some of the topics covered here, enroll in the `nRF Connect SDK Intermediate course`_ in the `Nordic Developer Academy`_.
```

Comment:
This should be a note, like the similar line on https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/installation.html .

Also, can we have this update as a separate commit in this PR? It is not really related to the release notes.

Reply:
Agreed on this being a note.

Reply:
Changed to a note